otx/algorithms/segmentation/configs/ocr_lite_hrnet_18_mod2/pot_optimization_config.json

```diff
@@ -5,7 +5,6 @@
       "params": {
         "preset": "mixed",
         "target_device": "ANY",
-        "use_fast_bias": false,
```

Comment:
Is this parameter true by default?

Reply:
Yes, default bias correction algorithm for DefaultQuantization is FastBiasCorrection